charts/lily/templates/daemon/app.yaml

```diff
@@ -0,0 +1,109 @@
+---
```

Comment:
How does this differ from the `worker` and `notifier` configs?

Reply:
I've refactored the configurations into reused templates in https://github.com/filecoin-project/helm-charts/pull/160/commits/16c350388d7cbc8e53733c825fcf28177d673014